test/addons-abi/test_instanceof/test2.js

```diff
@@ -0,0 +1,4 @@
+var addon = require( "./build/Release/test_instanceof" );
+function F() {}
+var x = 1;
+addon.doInstanceOf( x, F );
```

Comment:
I don't understand the purpose of this test file. It doesn't actually assert anything.